apps/coscrad-frontend/src/components/resources/playlists/playlist-index.presenter.tsx

```diff
@@ -0,0 +1,32 @@
+import { IPlayListViewModel } from '@coscrad/api-interfaces';
+import { PlaylistIndexState } from '../../../store/slices/resources/playlists/types';
+import { HeadingLabel, IndexTable } from '../../../utils/generic-components/presenters/tables';
+import { renderAggregateIdCell } from '../utils/render-aggregate-id-cell';
+
+import { CellRenderersDefinition } from '../../../utils/generic-components/presenters/tables/generic-index-table-presenter/types/cell-renderers-definition';
+import { renderMediaLengthInSeconds } from '../utils/render-media-length-in-seconds-cell';
+import { renderMultilingualTextCell } from '../utils/render-multilingual-text-cell';
+
+export const PlaylistIndexPresenter = ({ entities: playlists }: PlaylistIndexState) => {
+    const headingLabels: HeadingLabel<IPlayListViewModel>[] = [
+        { propertyKey: 'id', headingLabel: 'Link' },
+        { propertyKey: 'episodes', headingLabel: 'Number of Episodes' },
+        { propertyKey: 'name', headingLabel: 'Playlist' },
+    ];
+
+    const cellRenderersDefinition: CellRenderersDefinition<IPlayListViewModel> = {
+        id: renderAggregateIdCell,
+        name: ({ name }) => renderMultilingualTextCell(name),
+        episodes: ({ episodes }) => renderMediaLengthInSeconds(episodes.length),
```

Comment:
You shouldn't wrap `renderMediaLengthInSeconds`. This is the length of an array, i.e. the number of episodes in total. It is not a length of time.
```ts
({ episodes }) => episodes.length.toString()
```
should work here.

Reply:
resolved